### PR TITLE
Launch <amp-action-macro>

### PR DESCRIPTION
**Pull request by alabiaga**
cc\ @sebastianbenz

### CODE REVIEW

File: tools/experiments/experiments.js [hunk 1]
```diff
@@ -163,12 +163,6 @@ const EXPERIMENTS = [
     spec: 'https://github.com/ampproject/amphtml/issues/6106',
     cleanupIssue: 'https://github.com/ampproject/amphtml/pull/6351',
   },
-  {
-    id: 'amp-action-macro',
-    name: 'AMP extension for defining action macros',
-    spec: 'https://github.com/ampproject/amphtml/issues/19494',
-    cleanupIssue: 'https://github.com/ampproject/amphtml/pull/19495',
-  },
```
Comment (choumx) [inline tools/experiments/experiments.js]: This just removes it from the list on the experiments page. To actually launch the extension, you have to add `"amp-action-macro: 1` to prod-config.json and canary-config.json. I'd revert this change.

**Review by dreamofabear — CHANGES_REQUESTED** [review]

File: tools/experiments/experiments.js (hunk 1 above)
Comment (alabiaga) [reply tools/experiments/experiments.js]: done. thanks

**Comment by choumx** [comment]
/to @ampproject/wg-approvers Seeking approval for a new extension launch. Documentation here: https://github.com/ampproject/amphtml/blob/master/extensions/amp-action-macro/amp-action-macro.md

**Review by dreamofabear — COMMENTED** [review]
We should also remove the "experimental" status from amp-action-macro.md.

**Comment by cramforce** [comment]
Oh, this hasn't launched? I have no objections.

**Review by dreamofabear — APPROVED** [review]

File: build-system/global-configs/prod-config.json [hunk 2]
```diff
@@ -20,6 +20,7 @@
 
   "a4aProfilingRate": 0.01,
   "amp-access-iframe": 1,
+  "amp-action-macro": 1,
```
Comment (msoo14131) [inline build-system/global-configs/prod-config.json:23]: 23

**Review by firoj0 — CHANGES_REQUESTED** [review]
Hello world

**Review by msoo14131 — COMMENTED** [review]
name: _
publish_to: none
environment:
  sdk: ^3.6.0
workspace:
  - packages/helper
  - packages/client_package
  - packages/server_package

File: build-system/global-configs/prod-config.json (hunk 2 above)
Comment (oga35767-eng) [reply build-system/global-configs/prod-config.json:23]: Yy

Comment (oga35767-eng) [inline build-system/global-configs/prod-config.json:23]: 🎼

**Review by oga35767-eng — APPROVED** [review]

File: tools/experiments/experiments.js (hunk 1 above)
Comment (oga35767-eng) [reply tools/experiments/experiments.js]: 👍
